api/src/test/java/org/openmrs/util/OpenmrsUtilUnitTest.java

```diff
@@ -137,4 +147,128 @@ public void join_shouldReturnListElementsJoinedBySeparatorIfGivenSeparatorIsNull
 
 		assertThat(OpenmrsUtil.join(moduleList, null), is("module1nullmodule2"));
 	}
+
+	@Test
+	public void isStringInArray_shouldPassIfStringIsInArray() {
+
+		String[] array = new String[]{"element1","element2","element3"};
+
+		assertTrue(OpenmrsUtil.isStringInArray("element1", array));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldNotAllowNullValuesAsLowerAndHigherNormal() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(null);
```

Comment:
please add the test for the case of `LowNormal` being null

->

`isInNormalNumericRange_shouldReturnFalseIfLowNormalIsNull`